neofetch

```diff
@@ -999,6 +999,14 @@ get_distro() {
                     *) distro="OS Elbrus $(< /etc/mcst_version)"
                 esac
 
+            elif [[ -f /boot/dietpi/.version ]]; then
+                . /boot/dietpi/.version
+                case $distro_shorthand in
+                    on) distro="DietPi v$G_DIETPI_VERSION_CORE.$G_DIETPI_VERSION_SUB.$G_DIETPI_VERSION_RC" ;;
+                    tiny) distro='DietPi' ;;
+                    *) distro="DietPi v$G_DIETPI_VERSION_CORE.$G_DIETPI_VERSION_SUB.$G_DIETPI_VERSION_RC ($G_GITOWNER/$G_GITBRANCH)"
+                esac
+
```

Comment:
The existing detection methods don't work?

Reply:
DietPi is based on Debian/Raspbian, and shows up as such.  It's really just a bunch of scripts and optimizations on top of that, and not a full distribution.

Reply:
If it's not a full distribution, it shouldn't be added to neofetch (line has to be drawn somewhere). Thanks for the PR.